software/earl/authenticator.go

```diff
@@ -502,7 +502,7 @@ func (a *FileBasedAuthenticator) userHasAccess(user *User, target Target) (AuthR
 		if !isday {
 			return AuthOkButOutsideTime,
 				fmt.Sprintf("Fulltime user outside %d:00..%d:00",
-					hour_from, hour_to)
+					time_from, time_to)
```

Comment:
This should do the time_from.fmt("15:04") thing, not the %d which attempts to print an integer.